lib/AFS/CellCC/Util.pm

```diff
@@ -340,6 +340,68 @@ scratch_ok($$$$$) {
     return 1;
 }
 
+# Same as scratch_ok but receives several jobs. The destination cells of the
+# jobs received as an argument are requesting the same volume. Knowing that, we
+# dump the volume only once and create hardlinks for the jobs. Since the size
+# of the volume should be much greater than the size of the hardlinks, and
+# assuming we should not have that many destination cells, the size of the
+# hardlinks are not taken into consideration.
+sub
+scratch_ok_jobs($$$$@) {
```

Comment:
having a separate function for this seems really unnecessary. just make scratch_ok take an array of jobs? and if there are still existing callers that only use 1 job, just pass an array only containing that 1 job